### PR TITLE
📚 Add license to source files

### PR DESCRIPTION
**Pull request by laurentsimon**
closes https://github.com/gossts/slsa-go/issues/9

### CODE REVIEW

File: .github/workflows/builder.yml [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2021 The GOSST team.
```
Comment (asraa) [inline .github/workflows/builder.yml:1]: is it 2022? hahahhaa scary

**Review by asraa — CHANGES_REQUESTED** [review]

File: .github/workflows/builder.yml (hunk 1 above)
Comment (laurentsimon) [reply .github/workflows/builder.yml:1]: Let me remove the year... apparently it's not even necessary to have it

Comment (asraa) [inline .github/workflows/builder.yml:1]: just missing this one?

Comment (laurentsimon) [reply .github/workflows/builder.yml]: done. sorry, my find command was only looking for *.go :/ my bad

**Review by asraa — APPROVED** [review]

**Pull request merged**